x/circuit/features/msg_trip.feature

```diff
@@ -0,0 +1,66 @@
+Feature: MsgTripCircuitBreaker
+	Circuit breaker can disable message execution:
+	- when the caller trips the circuitbreaker for a message(s)
+	- when the caller has the correct permissions
+
+  Rule: a user must have permission to trip the circuit breaker for a message(s)
```

Comment:
This looks okay as a rule. For `msg_reset` I suggest either one rule like this or one rule for each of the different permissions types (as I've suggested above). The idea is that a rule describes succinctly what _must_ be true